runtime/CMakeLists.txt

```diff
@@ -148,6 +148,9 @@ if(J9VM_IS_NON_STAGING)
 else()
 	# if we aren't generating nls headers, just create a dummy target
 	add_custom_target(j9vm_nlsgen)
+
+	# we also need a dummy target for run_cptool
+	add_custom_target(run_cptool)
```

Comment:
Why is defined here instead of in `runtime/jcl/CMakeLists.txt `?

Reply:
No particular reason, but now that you mention it `runtime/jcl/CMakeLists.txt` makes more sense